aries/aries_vcx_core/src/wallet/indy/mod.rs

```diff
@@ -27,119 +57,91 @@ impl IndySdkWallet {
     }
 }
 
-#[derive(Clone, Debug, TypedBuilder, Serialize, Deserialize)]
-#[builder(field_defaults(default))]
-pub struct WalletConfig {
-    pub wallet_name: String,
-    pub wallet_key: String,
-    pub wallet_key_derivation: String,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    #[builder(setter(strip_option))]
-    pub wallet_type: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    #[builder(setter(strip_option))]
-    pub storage_config: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    #[builder(setter(strip_option))]
-    pub storage_credentials: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    #[builder(setter(strip_option))]
-    pub rekey: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    #[builder(setter(strip_option))]
-    pub rekey_derivation_method: Option<String>,
-}
+const WALLET_OPTIONS: &str =
+    r#"{"retrieveType": true, "retrieveValue": true, "retrieveTags": true}"#;
 
-#[derive(Clone, Debug, TypedBuilder, Serialize, Deserialize)]
-#[builder(field_defaults(default))]
-pub struct IssuerConfig {
-    pub institution_did: String,
-}
+const SEARCH_OPTIONS: &str = r#"{"retrieveType": true, "retrieveValue": true, "retrieveTags": true, "retrieveRecords": true}"#;
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
-struct WalletCredentials {
-    key: String,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    rekey: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    storage_credentials: Option<serde_json::Value>,
-    key_derivation_method: String,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    rekey_derivation_method: Option<String>,
-}
+#[async_trait]
+impl BaseWallet for IndySdkWallet {
```

Comment:
Why is it called `IndySdkWallet`, while the feature flag is called `vdrtools_wallet`? Doesn't it seem wrong? A newcomer would be rightfully confused.

Reply:
Having unified naming would be nice, any suggestions? 

However these are existing names and a rename could be done separately.

Reply:
What suggestion do you have in mind? vdrtools vs. indy-sdk? Do *you* have an opinion?

Reply:
I am asking about your suggestion for a new name since you brought it up.

We could rename the struct to `VdrtoolsWallet` or change to `IndyWallet` and change the flag to `indy_wallet` as well.

Reply:
I agree renaming/unifying can be done, but possibly not a concern of this PR. There was an inconsistency between indysdk/vdrtools naming already before. I'll leave to your judgement @xprazak2 to address this / or not / in this PR